chat-commands.js

```diff
@@ -816,6 +826,7 @@ exports.commands = {
 			}
 		} else {
 			const settingName = (setting === true ? 'secret' : setting);
+			if (room.subRooms && room.subRooms.size) return this.errorReply("Private rooms cannot have subrooms.");
```

Comment:
I don't see a problem with this, either?

Reply:
I decided this was not a can of worms I wanted to open at the time. Maybe eventually if a policy and usecase is more fleshed out, but didn't want to bother atm.

Reply:
...acceptable.